index.html

```diff
@@ -199,14 +199,15 @@ <h2>About me</h2>
         </section>
         <section class="form">
             <div class="formflex">
-                <p>I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like to chat please<br>get in touch.</p>
+                <h2>I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like to chat please<br>get in touch.</h2>
             <form>
-                <input class="name1" type="text" placeholder="Full Name">
+                <input class="name1" type="text" placeholder="Full Name" maxlength="30">
                 <input class="name" type="text" placeholder="First Name">
                 <input class="name" type="text" placeholder="Last Name"><br>
-                <input class="email" type="text" placeholder="Email address"><br>
-                <input class="note" type="text" placeholder="Hello Mariane. Glad to see you!"><br>
-                <button type="submit">Get in touch</button>
+                <input class="email" type="email" placeholder="Email address"><br>
+                <textarea class="note" placeholder="Hello Femi. Glad to see you!" maxlength="500"></textarea><br>
```

Comment:
- [ ] Please all inputs should have the `required` attribute to activate the validation on submit.